strabo/database.py

```diff
@@ -35,6 +35,19 @@ def delete_image(id):
     idquery.delete()
     db.session.commit()
 
+
+def delete_unrefrenced_images(old_images,images):
+    '''deletes images in old_images which are not in images'''
+    for old_img in old_images:
+        if all(old_img.id != img.id for img in images):
+            delete_image(old_img.id)
+
+def jsonifyable_row(sql_row):
```

Comment:
sqlachemy rows cannot be turned into json strings in their original form. So it is converted into a dictionary of column names to data, which is convertible to json strings, so it can easily be sent over the web to the javascript, where it is turned back into an object that corresponds exactly to the original row object.